config/Config.cfc

```diff
@@ -65,6 +65,21 @@ component {
 			, title         = "cms:sitenav.managesites"
 			, icon          = "fa-globe"
 		};
+
+		settings.adminConfigurationMenuItems = settings.adminConfigurationMenuItems ?: [];
+
+		ArrayAppend( settings.adminConfigurationMenuItems, "adminManager" );
+
+		settings.adminMenuItems = settings.adminMenuItems ?: {};
+
+		settings.adminMenuItems.adminManager = {
+			  buildLinkArgs = { linkTo="adminManager.users" }
+			, activeChecks  = { handlerPatterns="^admin\.adminManager\.users" }
+			, permissionKey = "websiteUserManager.navigate"
```

Comment:
This is the wrong permission key - should be related to admin users, rather than website users.

Reply:
Updated to usermanager